src/test/java/com/synopsys/integration/alert/web/api/authentication/AuthenticationControllerTestIT.java

```diff
@@ -87,110 +84,49 @@ public void testLogin() throws Exception {
         mockMvc.perform(request).andExpect(MockMvcResultMatchers.status().isNoContent());
     }
 
-    @Test
-    public void userLogoutWithValidSessionTest() {
-        AuthenticationController loginHandler = new AuthenticationController(null, null, csrfTokenRepository);
-        MockHttpServletRequest request = new MockHttpServletRequest();
-        MockHttpServletResponse response = new MockHttpServletResponse();
-        MockHttpSession session = (MockHttpSession) request.getSession(true);
-        session.setMaxInactiveInterval(30);
-        loginHandler.logout(request, response);
-        assertTrue(session.isInvalid(), "Expected the session to be invalid");
-        assertTrue(response.containsHeader("Location"), "Expected the response to contain a Location header");
-    }
-
-    @Test
-    public void userLogoutWithInvalidSessionTest() {
-        AuthenticationController loginHandler = new AuthenticationController(null, null, csrfTokenRepository);
-        HttpServletRequest request = new MockHttpServletRequest();
-        MockHttpServletResponse response = new MockHttpServletResponse();
-        loginHandler.logout(request, response);
-    }
-
     @Test
     public void userLoginWithValidSessionTest() {
-        LoginActions loginActions = Mockito.mock(LoginActions.class);
-        AuthenticationController loginHandler = new AuthenticationController(loginActions, null, csrfTokenRepository);
+        AuthenticationActions authenticationActions = Mockito.mock(AuthenticationActions.class);
```

Comment:
Kept the tests that deal with sessions here since controllers manage valid and invalid HttpSession objects.